### PR TITLE
Globally initialize tooltips

### PR DESCRIPTION
**Pull request by nwalters512**
This is in the same spirit as #10394.

### CODE REVIEW

File: apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts [hunk 1]
```diff
@@ -95,7 +95,11 @@ onDocumentReady(() => {
     onCheckAll: updateGradingTagButton,
     onCheckSome: updateGradingTagButton,
     onCreatedControls: () => {
-      $('#grading-table th[data-field="points"] .filter-control input').tooltip({
+      $('#grading-table th[data-field="auto_points"] .filter-control input').tooltip({
+        title: 'hint: use <code>&lt;1</code>, or <code>&gt;0</code>',
+        html: true,
+      });
+      $('#grading-table th[data-field="manual_points"] .filter-control input').tooltip({
```
Comment (nwalters512) [inline apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts:102]: AFAICT this table predates the manual/auto points split, as there is no longer a `points` column. This update accounts for the separate columns.

File: apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts [hunk 2]
```diff
@@ -106,14 +110,12 @@ onDocumentReady(() => {
...
     onPreBody: () => {
       $('#grading-table [data-toggle="popover"]').popover('dispose');
-      $('#grading-table [data-toggle="tooltip"]').tooltip('dispose');
     },
     onPostBody: () => {
       updateGradingTagButton();
       $('#grading-table [data-toggle="popover"]')
         .popover({ sanitize: false })
         .on('shown.bs.popover', updatePointsPopoverHandlers);
-      $('#grading-table [data-toggle=tooltip]').tooltip({ html: true });
```
Comment (nwalters512) [inline apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts:116]: AFAICT the `html: true` was unnecessary. There's a single tooltip (open instance warning) and it doesn't use HTML in the tooltip content.

File: apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts (hunk 1 above)
Comment (jonatanschroeder) [reply apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts:102]: The column still exists, it's just hidden by default and can be shown by selecting "Columns".
https://github.com/PrairieLearn/PrairieLearn/blob/ed58c53f1557719f1f2f2c0a65c7903e37186e15/apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts#L207-L217

Comment (nwalters512) [reply apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts:102]: Huh, sure enough. Is there a reason you left the tooltip off the other columns? Or is that change fine, and I should just add it back to `points`?

Comment (jonatanschroeder) [reply apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts:102]: I checked the history, indeed it predates the split of manual/auto points. I'm guessing I forgot to add the tooltip [when adding the new columns](https://github.com/PrairieLearn/PrairieLearn/pull/5909/files#diff-d89e0a400fcf53add728429e674fea25a80061668ac2b5955526104e85aa88c8).

Comment (nwalters512) [reply apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts]: Added back in 1b84674.

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**